larray_editor/start.py

```diff
@@ -9,7 +9,7 @@ def main():
     if os.name == 'nt':
         sys.stdout = open(os.devnull, "w")
         sys.stderr = open(os.path.join(os.getenv("TEMP"), "stderr-" + os.path.basename(sys.argv[0])), "w")
-    edit(*sys.argv[1:])
```

Comment:
you need to keep sys.argv, to be able to specify a file from the command line (or to bind an extension like .h5 to be opened by the editor)